ROADMAP.md

```diff
@@ -0,0 +1,21 @@
+# Roadmap
```

Comment:
Can you pull the highlights from here and place as checkboxes on the main README?